src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -2477,6 +2517,16 @@ bool clang_c_convertert::get_cast_expr(
 
   case clang::CK_VectorSplat:
     break;
+
+#ifdef ESBMC_CHERI_CLANG
+  case clang::CK_PointerToCHERICapability:
```

Comment:
What is the purpose of these cases?

Reply:
The purpose is to track whether these constants change their name in upstream CHERI-Clang and to remind me to translate these conversions properly in a later PR.